packages/lib/src/interactions/models.ts

```diff
@@ -36,7 +36,7 @@ export interface InteractionInfo {
 
 export interface Interaction {
   button?: MouseButton;
-  modifierKey?: ModifierKey;
+  modifierKeys?: ModifierKey[];
```

Comment:
The change in essence: `Interaction` can now specify multiple `modifierKeys`

The rest of the PR is adaptation to this change